File: app/src/main/java/com/example/yourapp/MainActivity.java
package com.example.yourapp;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        setTheme(R.style.AppTheme); 
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        // ...existing code...
    }
}

File: app/src/main/java/com/example/yourapp/MainActivity.kt
package com.example.yourapp

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(R.style.AppTheme) // Set your app's theme here
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        // ...existing code...
    }
}
